CHANGELOG.md

```diff
@@ -83,6 +83,7 @@ Graph:
 
 - Installation of parser when installing RecordFlux from PyPI (#745)
 - Examples in README (#879)
+- Support for newer GNAT versions (#1116)
```

Comment:
We already have "Compatibility of generated code to FSF GNAT 11 and GNAT Pro 23" under "Added". I wouldn't consider adding support for a previously not supported compiler version as a bug fix.

Reply:
Fixed.